lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb

```diff
@@ -1,19 +0,0 @@
-module DataUpdateScripts
```

Comment:
Given our recent discussions on forem.team, maybe this shouldn't be removed at all? I'm happy to restore it.